acceptance-tests/src/test/java/linea/plugin/acc/test/BlobTransactionDenialTest.java

```diff
@@ -0,0 +1,141 @@
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package linea.plugin.acc.test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+import com.google.common.io.Resources;
+import org.apache.tuweni.bytes.Bytes;
+import org.hyperledger.besu.tests.acceptance.dsl.account.Accounts;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.web3j.crypto.Blob;
+import org.web3j.crypto.BlobUtils;
+import org.web3j.crypto.Credentials;
+import org.web3j.crypto.RawTransaction;
+import org.web3j.crypto.TransactionEncoder;
+import org.web3j.protocol.Web3j;
+import org.web3j.protocol.core.DefaultBlockParameterName;
+import org.web3j.protocol.core.methods.response.EthSendTransaction;
+import org.web3j.tx.RawTransactionManager;
+import org.web3j.tx.TransactionManager;
+import org.web3j.tx.gas.DefaultGasProvider;
+import org.web3j.utils.Numeric;
+
+/**
+ * Tests that verify the LineaPermissioningPlugin correctly rejects BLOB transactions while allowing
+ * other transaction types.
+ */
+public class BlobTransactionDenialTest extends LineaPluginTestBase {
+
+  private static final BigInteger GAS_PRICE = DefaultGasProvider.GAS_PRICE;
+  private static final BigInteger GAS_LIMIT = DefaultGasProvider.GAS_LIMIT;
+  private static final BigInteger VALUE = BigInteger.ZERO;
+  private static final String DATA = "0x";
+
+  private Web3j web3j;
+  private Credentials credentials;
+  private TransactionManager txManager;
+  private String recipient;
+
+  @Override
+  @BeforeEach
+  public void setup() throws Exception {
+    super.setup();
+    web3j = minerNode.nodeRequests().eth();
+    credentials = Credentials.create(Accounts.GENESIS_ACCOUNT_ONE_PRIVATE_KEY);
+    txManager = new RawTransactionManager(web3j, credentials, CHAIN_ID);
+    recipient = accounts.getSecondaryBenefactor().getAddress();
+  }
+
+  @Test
+  public void legacyTransactionsAreAccepted() throws Exception {
+    // Act - Send a legacy transaction
+    String txHash =
+        txManager
+            .sendTransaction(GAS_PRICE, GAS_LIMIT, recipient, DATA, VALUE)
+            .getTransactionHash();
+
+    // Assert
+    minerNode.verify(eth.expectSuccessfulTransactionReceipt(txHash));
+  }
+
+  @Test
+  public void eip1559TransactionsAreAccepted() throws Exception {
+    // Act - Send an EIP-1559 transaction
+    String txHash =
+        txManager
+            .sendEIP1559Transaction(
+                CHAIN_ID, GAS_PRICE, GAS_PRICE, GAS_LIMIT, recipient, DATA, VALUE)
+            .getTransactionHash();
+
+    // Assert
+    minerNode.verify(eth.expectSuccessfulTransactionReceipt(txHash));
+  }
+
+  // Note that this currently passes as Besu Node does not support blob tx
+  // It is not currently a test that the LineaPermissioningPlugin is blocking blob tx
+  @Test
```

Comment:
Can we create a separate testcase for Prague fork for this testcase specifically?

I know that eventually `LineaPluginTestBase` will migrate to Prague completely and we wouldn't have to handle this case separately, but it would be nice to properly test it to close this ticket now. We really don't want to miss this when we migrate to Prague.